ERCS/erc-7766.md

```diff
@@ -0,0 +1,166 @@
+---
+eip: 7766
```

Comment:
```suggestion
eip: 7765
```
Assigning next sequential EIP/ERC/RIP number.  Please don't assign your own number.

Please also update the filename.

Reply:
ok, will do it. If I don't give a EIP Number, the checks from the bot will not pass. So my EIP now uses the number 7765, right ?